test/test/helpers/use-nock/README.md

```diff
@@ -18,7 +18,7 @@ useNock();
 ```js
 import useNock, { nock } from 'test/helpers/use-nock';
 //or
-import { useNock, nock } from 'text/helpers/use-nock';
+import { useNock, nock } from 'test/helpers/use-nock';
```

Comment:
>I see that we can also expose nock from this helper, nice :+1: 

My observed downsides of this are that it confuses our distinction between external and internal dependencies, ties us to a particular implementation (vs. importing `nock` independently) if we were to move away from this helper or to another, and is a convenience made obvious only through knowledge of its existence (i.e. as an outside developer, it might take me a moment to understand how it's being imported).